### ISSUE TITLE
Quest Fortunate Misunderstandings

### ISSUE THREAD

**Issue by Leyfur** [dalaranwow/dalaran-wow#5027]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
I pick up the quest from the npc called "Injured Rainspeaker Oracle" in scholasar bazin. he tells me to let him know when i am ready but when i speak to him he gives me no option to start walking.

**Expected behaviour**: Tell us what should happen instead
The text box to tell him that i am ready to escort him should pop up and he should start walking to his village

**Steps to reproduce the problem**:

1. It happens all the time and not just to me
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

Link to quest : http://www.wowhead.com/quest=12570/fortunate-misunderstandings#comments
comment on said quest on wowhead : "so, what you have to do is simple, once you take the quest, remember to talk to the oracle again and choose to take it back home, once you do that, it will simple take its way which you have to follow and protect it till it goes to the camp, simple and easy"

**Include Screenshots from the issue if necessary**

**Comment by Lothloryen** [comment]
Hello there,

Thank you for your report, the issue that was preventing some players from doing this quest has been corrected, it will be applied to the live realm with the next scheduled maintenance.

It was doable, but there are some chances for the Injured Rainspeakers to get glitched and not send the gossip to start the escort.

Regards.
